package.json

```diff
@@ -22,7 +22,9 @@
     "lint:check": "lerna run lint:check --parallel --no-bail -- --max-warnings=0",
     "lint:write": "lerna run lint:write --parallel --no-bail",
     "publish": "lerna publish --yes from-package patch",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package",
+    "build:netlify": "lerna run --stream --parallel build:netlify",
+    "release:netlify": "lerna run --stream --parallel release:netlify -- --auth $NETLIFY_ACCESS_TOKEN --message $(git rev-parse --short HEAD) $(case $(git rev-parse --abbrev-ref HEAD) in (master) echo --prod;; esac)"
```

Comment:
Am I right in thinking this command will always run? Could we use the `--since` flag for `lerna` (as in L20) to skip the netlify release when that package didn't change?

Reply:
Yep you are right about that, good catch. We can probably skip `build:netlify` for unchanged packages as well.